src/game/server/neo/neo_message.cpp

```diff
@@ -43,12 +53,34 @@ int CNEO_Message::UpdateTransmitState()
     return SetTransmitState(FL_EDICT_ALWAYS);
 }
 
+void CNEO_Message::Think()
+{
+	DisplayTimer();
+	SetNextThink(gpGlobals->curtime + 0.05f);
```

Comment:
If we care about accurate completion time, then could this think delay introduce some inaccuracy? Maybe lower the think delay to `+ 0.01f` and then display only up to hundreths of a second for the time instead of milliseconds?